src/fileAPI.ts

```diff
@@ -40,16 +40,20 @@ export class FileAPI {
     return Math.ceil(numOfPages);
   }
 
-  myFiles = async (pageNumber: number = 1, pageSize: number = 20, appid : number) => {
+  myFiles = async (pageNumber: number = 1, pageSize: number = 20, appid? : number) => {
```

Comment:
take app id from constructor instead of functional param

Reply:
added appId as member in FileAPI